content/en/llm_observability/configuration/_index.md

```diff
@@ -15,7 +15,7 @@ further_reading:
 
 ## Overview
 
-You can configure your LLM applications on the [Configuration page][1] with settings that can optimize your application's performance and security. 
+You can configure evaluations for each of your LLM applications in LLM Observability
```

Comment:
missing a period at the end.

also you should delete `[1]: https://app.datadoghq.com/llm/config` at the bottom of the file, since we're not using that link anymore